Change SMB1360 configuration to match what we use for the mainline driver.
This makes it easier to compare the registers programmed by the downstream driver
with the mainline driver.

diff --git a/arch/arm/boot/dts/qcom/picmt.dts b/arch/arm/boot/dts/qcom/picmt.dts
index cfb53665..b9c82ed2 100644
--- a/arch/arm/boot/dts/qcom/picmt.dts
+++ b/arch/arm/boot/dts/qcom/picmt.dts
@@ -320,7 +320,7 @@
 			qcom,fg-batt-capacity-mah = <2810>;
 			qcom,fg-cc-soc-coeff = <0x81D5>;
 			qcom,fg-delta-soc = <1>;
-			qcom,fg-soc-min = <3>;
+			qcom,fg-soc-min = <15>;

 			qcom,fg-cutoff-voltage-mv = <3400>;
 			qcom,fg-iterm-ma = <100>;
@@ -330,14 +330,14 @@

 			qcom,float-voltage-mv = <4400>;
 			qcom,iterm-ma = <100>;
-			qcom,charging-timeout = <768>;
+			qcom,charging-timeout = <192>;
 			qcom,fg-reset-at-pon;
 			qcom,rsense-10mhom;
 			qcom,fg-reset-thresold-mv = <200>;
 			qcom,otg-batt-curr-limit = <1500>;
 			qcom,recharge-thresh-mv = <50>;
 			qcom,thermal-mitigation = <1500 700 600 0>;
-			qcom,fg-auto-recharge-soc = <97>;
+			qcom,fg-auto-recharge-soc = <95>;
 			qcom,empty-soc-disabled;
 			regulator-name = "smb1360_otg_vreg";
 			qcom,awake-min-soc;
